Sources/Moya/URLRequest+Encoding.swift

```diff
@@ -4,7 +4,6 @@ internal extension URLRequest {
 
     mutating func encoded(encodable: Encodable, encoder: JSONEncoder = JSONEncoder()) throws -> URLRequest {
         do {
-            let encodable = AnyEncodable(encodable)
```

Comment:
It seems the CI is failing because of this change.
Please have a look:
https://app.circleci.com/pipelines/github/Moya/Moya/350/workflows/6497f7db-7bee-47b4-b022-f893714bd2c5/jobs/4467?invite=true#step-106-2677

Reply:
thank you notice me👍🏻